ddsp/training/train_util_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The DDSP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for ddsp.training.test_util."""
+
+import json
+import os
+import tensorflow.compat.v2 as tf
+from tensorflow.python.eager import context
+import train_util
+
+
+class GetStrategyTest(tf.test.TestCase):
+
+  def test_multiworker_strategy_is_set(self):
+    """Check if proper strategy is returned and
+    that cluster configuration for multi worker strategy is set."""
```

Comment:
Short docstrings should fit on a single line.

Reply:
Done